capsules/src/rf233_const.rs

```diff
@@ -0,0 +1,150 @@
+#![allow(non_camel_case_types)]
+
+#[derive(PartialEq, Copy, Clone)]
```

Comment:
Could you add `Debug` here? It's nice to have that derived on all the basic enums